dev-lang/nqp/nqp-2023.02-r1.ebuild

```diff
@@ -5,35 +5,34 @@ EAPI=8
 
 inherit java-pkg-opt-2 multibuild
 
-if [[ ${PV} == "9999" ]]; then
-	EGIT_REPO_URI="https://github.com/perl6/${PN}.git"
-	inherit git-r3
-else
-	SRC_URI="https://github.com/perl6/${PN}/releases/download/${PV}/${P}.tar.gz"
-	KEYWORDS="~amd64 ~x86"
-fi
-
 DESCRIPTION="Not Quite Perl, a Raku bootstrapping compiler"
-HOMEPAGE="https://rakudo.org"
-
+HOMEPAGE="https://rakudo.org
+	https://github.com/Raku/nqp"
+SRC_URI="https://github.com/Raku/${PN}/releases/download/${PV}/${P}.tar.gz"
 LICENSE="Artistic-2"
 SLOT="0"
-IUSE="doc clang java +moar test"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc java +moar test"
 RESTRICT="!test? ( test )"
 REQUIRED_USE="|| ( java moar )"
 
-CDEPEND="java? (
-		dev-java/asm:9
+CDEPEND="
+	java? (
+		dev-java/asm:4
```

Comment:
dev-java/asm:4 is gone

Reply:
There is dev-java/asm:9.

I'd rather exclude java from rakudo because

1. I don't use java on my system. Java backend is not going to be tested by me.
2. I think JVM is a dead end.
3. rakudo is not going to shine on JVM. It should run on MoarVM.

Reply:
> Should we cut java out of rakudo?

Up to package maintainer, see https://gitweb.gentoo.org/repo/gentoo.git/commit/dev-lang/nqp?id=f94fd00ae0ca39fc553dd8e83f135c0004614e2b.

Reply:
Okay, I will just let him deal with java backend. I will update accordingly.

Reply:
Closing this to make a new pull request.